.gitlab-ci.yml

```diff
@@ -603,23 +603,24 @@ inttest:registryAuth:
     - echo "$DB_SERVICE_ACCOUNT_PRIVATE_KEY" > db-service-account-private-key.json
     - kubectl create secret generic storage-account-credentials --from-file db-service-account-private-key.json --namespace $CI_COMMIT_REF_SLUG --dry-run -o json | kubectl apply --namespace $CI_COMMIT_REF_SLUG -f -
     - kubectl create secret generic storage-secrets --from-literal=accesskey=$MINIO_ACCESS_KEY --from-literal=secretkey=$MINIO_SECRET_KEY --namespace $CI_COMMIT_REF_SLUG --dry-run -o json | kubectl apply --namespace $CI_COMMIT_REF_SLUG -f -
-
-    # Helm upgrade
-    - helm init --upgrade
+    - helm version
+    - helm repo add incubator http://storage.googleapis.com/kubernetes-charts-incubator
```

Comment:
You don't need to add repo here (or update the repo).
All dependencies are resolved (and downloaded) at `buildtest:helm-charts` step.
And helm charts files are passed to all deploy steps via artifacts.
You should make sure `buildtest:helm-charts` step run OK as that where the dependencies are resolved & downlaoded

Reply:
Yep, you are right. I was using the old image for that part. Removed the update repo and add repo